example/src/main/java/com/ryanharter/auto/value/moshi/example/BirthDateAdapter.java

```diff
@@ -0,0 +1,28 @@
+package com.ryanharter.auto.value.moshi.example;
+
+import com.squareup.moshi.FromJson;
+import com.squareup.moshi.ToJson;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Locale;
+
+public class BirthDateAdapter {
```

Comment:
Can you doc this?

Reply:
In particular - it seems a bit unnecessary and more of a Moshi example than an auto-value-moshi one

Reply:
ok. will remove this